modules/data/webadmin/tmpl/add_edit_user.tmpl

```diff
@@ -129,43 +129,47 @@
 		<? IF Edit ?>
 		<div class="sectionbg">
 			<div class="sectionbody">
-				<table class="sortable">
+				<? IF !NetworkEdit && !NetworkLoop ?>
+					<? FORMAT "No networks have been added for you yet." ?>
+				<? ELSE ?>
+				<table class="sortable networks" data-network-edit="<? IF NetworkEdit ?>1<? ELSE ?>0<? ENDIF ?>">
 					<thead>
 					<tr>
+						<td class="ignore-sort">
 						<? IF NetworkEdit ?>
-						<td class="ignore-sort">[<a href="<? VAR URIPrefix TOP ?><? VAR ModPath TOP ?>addnetwork?user=<? VAR Username ESC=URL ?>"><? FORMAT "Add" ?></a>]</td>
```

Comment:
Why did you remove ignore-sort?

Reply:
I restructured that table a bit to make it work properly if it's readonly.
The ignore-sort class is still there, see line 138.